docker-compose.test.yml

```diff
@@ -5,6 +5,7 @@ services:
     build:
       context: .
       dockerfile: dev/database/Dockerfile.centos
+    command: 'run-postgresql -c logging_collector=off'
```

Comment:
why do we add this?

Reply:
When running locally, we don't get any logs, since they are redirected to file by logging collector. This change will leave them on stderr instead of redirecting to file.

Reply:
aha, ok